src/osmf/lib/ExternalJavaScriptAPI.as

```diff
@@ -172,7 +174,7 @@ package
       var eventData : Object = new Object();
       eventData.eventtype = event.type;
       eventData.eventObject = event.eventObject;
-      SendToDebugger(eventData.eventtype, "onFlashEvent", "log");
+      //SendToDebugger(eventData.eventtype, "onFlashEvent", "log");
```

Comment:
SendToDebugger is our flash method to send to the console. This commented out one sends for every event - so was too spammy for a release, but I did not want to remove it as yet in case it was useful in debugging.